dev/tasks/dev/tasks/create-package.js

```diff
@@ -79,7 +80,7 @@ module.exports = ( ckeditor5Path, workspaceRoot ) => {
 			return inquiries.getPackageGitHubUrl( packageName );
```

Comment:
Better do it now, once we realised that it's incorrect. Also the inquiry message should be changed as I already commented in https://github.com/ckeditor/ckeditor5/pull/197#issuecomment-225570029.